src/content/snapshot/troubleshooting-snapshots.md

```diff
@@ -30,6 +30,45 @@ It's essential that your components and stories render in a **consistent** fashi
 - **Intentional randomness**: Some stories may render unpredictably intentionally. If this is the case you may want to [ignore the story](/docs/ignoring-elements) from UI Tests and move on.
   If you still need inconsistent elements for local development purposes inside Storybook, you can use `isChromatic()` exported from [our package](/docs/ischromatic) to apply the solutions above only when in the Chromatic environment.
 
+### Serve static files
+
+When using Playwright or Cypress, you can serve static files like fonts, images, and videos through your app server. This ensures that resources load consistently across all snapshots.
+
+For Storybook, use the [staticDirs](https://storybook.js.org/docs/configure/integration/images-and-assets#serving-static-files-via-storybook-configuration) option to load static files for your stories.
+
+### Use fixed-height wrappers for portal components
+
+Portals allow components to render arbitrary elements outside the parent component's initial DOM hierarchy. For example, tooltips, modals, popovers, and menus can be triggered by a nested button, but render close to the top of the DOM hierarchy using portals.
+
+With Playwright and Cypress, Chromatic snapshots the entire page, including modals, dialogs, and other portaled elements. However, with Storybook, portals render outside of Storybook’s DOM tree which can lead to cut-off snapshots.
+
+<details>
+<summary>Why are snapshots of portal components (tooltip, modal, popover, menu) cut off?</summary>
+
+For stories, Chromatic relies on the "natural" height of your component's outermost DOM element (using Storybook's `#storybook-root` element in version 7 or higher, or the `#root` element for previous versions) to determine snapshot dimensions. As portals render outside of Storybook's DOM tree, their dimensions cannot be auto-detected by Chromatic, which can lead to cut-off snapshots.
+
+</details>
+
+To capture snapshots of portaled elements, you can use a [decorator](https://storybook.js.org/docs/writing-stories/decorators#component-decorators) that wraps your stories in a fixed height container. You can adjust the height of the container to account for the total dimensions of your component and portal.
```

Comment:
Small item for your consideration when you can, feel free to disregard it
```suggestion
To capture snapshots of portaled elements, you can use a [decorator](https://storybook.js.org/docs/writing-stories/decorators#component-decorators) that wraps your stories in a fixed-height container. You can adjust the container's height to account for the total dimensions of your component and portal.
```

Also, now that we have the title option available for the code snippets, we can use it, and if you're ok with it, convert it to TypeScript to align with the rest of the documentation, as we already have moved in that direction. Wouldn't you agree?